SPECS/openssl/openssl.spec

```diff
@@ -9,7 +9,7 @@
 Summary: Utilities from the general purpose cryptography library with TLS implementation
 Name: openssl
 Version: 3.3.0
-Release: 2%{?dist}
+Release: 3%{?dist}
```

Comment:
I'm not sure I understand this part. OpenSSL doesn't build with SymCrypt as far as I could tell, so the bump shouldn't make a difference.

Reply:
This bump is part of this comment - https://github.com/microsoft/azurelinux/pull/10075#discussion_r1712170768

Reply:
Looks like I misunderstood the comment. Will unbump these.